apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -498,39 +511,87 @@ public Flow(long maxByteCount, long maxElementCount, Action<Task> registerTaskFn
             /// has completed.
             /// </summary>
             /// <param name="byteCount">The number of bytes in the element associated with <paramref name="fn"/>.</param>
+            /// <param name="orderingKey">The ordering key for this message. Empty string if no ordering key.</param>
             /// <param name="fn">The function to execute.</param>
             /// <returns>A Task that completes once <paramref name="fn"/> has been scheduled for execution.</returns>
-            public async Task Process(long byteCount, Func<Task> fn)
+            public async Task Process(long byteCount, string orderingKey, Func<Task> fn)
```

Comment:
It would be nice if this were internal - I hadn't spotted that it was in a private class for a bit, and worried that it would be a breaking change.

Reply:
All changed to `internal`.